### ISSUE TITLE
creating ID does not auto login

### ISSUE THREAD

**Issue by jimzucker** [capitalone/Hygieia#652]
I think this was always like this but when you create a new id, it says ID created but doesn't log you in, the behavior is odd, you have to figure out how to get to the login screen.

**Comment by sthatip** [comment]
It is not so odd because the signup may or may not be successful. But there is a login link on the signup page that you could always use. just need to enter your credentials again.

**Comment by jimzucker** [comment]
Poor choice of words. Let me rephrase.

Common patterns after creating an ID is successfully created are:
1. Return to the login screen (Obvious to user what to do)
2. Autologin 

Currently we stay at the create id screen which is not useful to me once I have an ID, so why leave me there.  There are use cases where people want to create multiple ids, but normally you have 2 buttons 'Create', 'Create another Id' something like that.

**Comment by satishc1** [comment]
@Sbrenthughes @tabladrum Please take a look at this one. Lets do the patten 1 in the suggestion above... return to the login screen once signup is sucessfully completed. Then the user can choose to login with the id they created.

**Comment by satishc1** [comment]
https://github.com/capitalone/Hygieia/pull/1129 fixes this one. When you register a user, it auto logs you in as that user.
